cloudhub-2/modules/ROOT/pages/_partials/deployment.adoc

```diff
@@ -96,6 +96,29 @@ To configure additional log levels and categories to include in logs:
 // end::configureLogging[]
 
 
+// tag::enableDistributedTracing[]
+
+== Enable Trace Data Collection
```

Comment:
Since Traces is different from Logging, it should not be under the "`Configure Logging for the App`" section.

Can we either elevate Traces to a new section, OR we change the broader section to "`Configure Monitoring for the App`"

Reply:
Changed the traces section to H2 so it shows as a separate section from "Configure Logging..."